lib/auth/auth.go

```diff
@@ -390,6 +398,13 @@ func NewServer(cfg *InitConfig, opts ...ServerOption) (*Server, error) {
 		)
 	}
 
+	go func() {
+		for {
+			as.CheckOSSDesktopsLimit(context.Background())
```

Comment:
If this fails we will never know. We should probably at least log the error.

Reply:
I've added logging.